docs/server/production-deploy.md

```diff
@@ -68,20 +68,30 @@ Every shard is low contention by design and it is very difficult to oversubscrib
 With that said, here are some guidelines to some common bottlenecks:
 
 - **Database**. The vast majority of the time the database will be the bottleneck. **We highly recommend setting alerts on `ScheduleToStart` latency** to look out for this. Also check if your database connection getting saturated.
-- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)). Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
+- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)).
+  Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
+  If you need more instances of each service, spin them up separately with different command line arguments. You can learn more cross referencing [our Helm chart](https://github.com/temporalio/helm-charts) with our [Server Configuration reference](https://docs.temporal.io/docs/server/configuration/).
```

Comment:
As per @mfateev comment on this [Slack Thread](https://temporalio.slack.com/archives/CTRCR8RBP/p1628981004218200): 

> They are all the same binary. So to spin up them separately you pass different command line arguments.

It would be good to mention that Frontend, Matching, History and Worker are bundled in one single binary, since the part 

>[...]the 4 *internal* services of Temporal [...]

is only implying it.